target/scripts/helpers/error.sh

```diff
@@ -62,13 +63,13 @@ function dms_panic__no_file { dms_panic 'no-file' "${1}" "${2}"; }
 function dms_panic__misconfigured { dms_panic 'misconfigured' "${1}" "${2}"; }
 function dms_panic__invalid_value { dms_panic 'invalid-value' "${1}" "${2}"; }
 
-# Call this method when you want to panic (emit a 'FATAL' log level error, and exit uncleanly).
+# Call this method when you want to panic (i.e. emit an 'ERROR' log and exit DMS with a non-zero exit code).
```

Comment:
I'm not sure how the change at the end is an improvement.

It's not any `non-zero` exit code, it's specifically `1`, no variability there for other values. We're performing `kill 1`, which to my knowledge is not a graceful shutdown, it's an unclean shutdown/exit is it not? If you agree, I think "exit uncleanly" communicates that better, the reader of the comment can see that it uses `exit 1`, is that adjustment to mention "a non-zero exit code" providing much value/context?

Reply:
I will revert it back.